nixos/modules/services/backup/restic.nix

```diff
@@ -132,7 +151,7 @@ with lib;
           restartIfChanged = false;
           serviceConfig = {
             Type = "oneshot";
-            ExecStart = "${resticCmd} backup ${concatStringsSep " " backup.extraBackupArgs} ${concatStringsSep " " backup.paths}";
+            ExecStart = [ "${pkgs.bash}/bin/bash -c '${resticCmd} backup ${concatStringsSep " " backup.extraBackupArgs} ${concatStringsSep " " backup.paths}'" ] ++ pruneCmd;
```

Comment:
Rather then bash, it should be possible to pass a list of commands to `ExecStart`

Reply:
I agree it's a bit wonky.  But I couldn't see another way to achieve my use case, which is dynamically finding all the `.git` folders I want to back up.  So my nix expression has:

```
      paths = [ "/home/matt/.bitcoin/wallet.dat" ];
      extraBackupArgs = [ "-v" "--files-from <(find /home/matt/git -type d -name .git)" ];
```

... which produces:

```
ExecStart=/nix/store/qckz...-bash-4.4-p19/bin/bash -c '/nix/store/griv...-restic-0.9.0-bin/bin/restic backup -v --files-from <(find /home/matt/git -type d -name .git) /home/matt/.bitcoin/wallet.dat'
```

If I roll my own `ExecStart` line then I have to reproduce the `${extraOptions}` handling (which is admittedly pretty easy ...).

Reply:
If it is enabled by default, we should use `script` instead of `ExecStart`, which automatically wraps it in a bash script.